packages/desktop/src/services/PersistentConfigServiceElectron.ts

```diff
@@ -139,15 +132,32 @@ export default class PersistentConfigServiceElectron implements PersistentConfig
     }
 
     public getAll(): PersistedConfig {
+        // Initialize default values
+        const defaultConfig: PersistedConfig = {
```

Comment:
Do we need to add this default here? Shouldn't all values be optional since a first-time user wouldn't have any of these values?

Reply:
restored to no defaults in [38e7ac2](https://github.com/AllenInstitute/biofile-finder/pull/223/commits/38e7ac251efb44254d2eb9eee39389e2b01ecc1b)